config/database.yml

```diff
@@ -21,8 +21,8 @@ default: &default
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   host: <%= ENV.fetch("DATABASE_HOST") { "localhost" } %>
-  username: <%= ENV.fetch("DATABASE_USERNAME") { "" } %>
-  password: <%= ENV.fetch("DATABASE_PASSWORD") { "" } %>
+  username: <%= ENV.fetch("DATABASE_USERNAME") { "victorortega" } %>
```

Comment:
this should be removed as well as the password...

Reply:
Changes done.